src/shared/components/copyable-text/CopyableText.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Button, ButtonVariant, Tooltip } from '@patternfly/react-core';
+import { CopyIcon } from '@patternfly/react-icons/dist/esm/icons/copy-icon';
+import './CopyableText.css';
+
+type CopyableTextProps = {
+  text: string;
+  'data-test'?: string;
+};
+
+/**
+ * CopyableText component displays text with a copy button
+ * Used for displaying image URLs that should be copied instead of clicked
+ */
+export const CopyableText: React.FC<CopyableTextProps> = ({ text, 'data-test': dataTest }) => {
```

Comment:
use ClipboardCopy component from patternfly instead of creating a new one https://v5-archive.patternfly.org/components/clipboard-copy#inline-compact-in-sentence.